.github/actions/branch/action.yml

```diff
@@ -17,4 +17,4 @@ runs:
       env:
         BRANCH_NAME: ${{ inputs.branch }}
       run: echo "BRANCH=$BRANCH_NAME" >> $GITHUB_ENV
-      shell: bash -ileo pipefail {0}
+      shell: bash --login -eo pipefail {0}
```

Comment:
I replaced `-l` with `--login` because I think using long param names in scripts helps with understandability. (`-e` and `-o` don't have long versions)